lib/moog.js

```diff
@@ -192,17 +192,24 @@ module.exports = function(options) {
           }
           if (properties.group) {
             const groups = klona(that[`${cascade}Groups`]);
-            for (const value of Object.values(properties.group)) {
+            for (const [ key, value ] of Object.entries(properties.group)) {
               for (const field of value.fields || []) {
-                for (const value of Object.values(groups)) {
```

Comment:
This kind of thing makes me cross eyed (the same constant name in a nested loop).